### PR TITLE
Add `filetype` and `filesize` to SMK script

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Related to #536 by @obulat 

## Description

This PR adds `filetype` and `filesize` to the list of data collected from the SMK API.

## Testing Instructions

The tests for the script should pass.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: openverse_catalog/dags/common/extensions.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from typing import Literal
+
+
+EXTENSIONS = {
+    "image": ["jpg", "jpeg", "png", "gif", "bmp", "webp", "tiff", "tif", "svg", "webp"],
+    "audio": ["mp3", "ogg", "wav", "aiff", "flac", "aac", "m4a", "wma", "mp4", "m4b"],
+}
+
+
+def extract_filetype(url: str, media_type: Literal["image", "audio"]) -> str | None:
```
Comment (AetherUnbound) [inline openverse_catalog/dags/common/extensions.py:10]: We had a [lengthy discussion](https://github.com/WordPress/openverse-catalog/pull/496#discussion_r868573848) about this `Literal` in a different PR 😅 I think we should probably just throw it in `constants.py` and use it from there! Even if we have to repeat ourselves, better than defining this everywhere we need it

**Review by AetherUnbound — COMMENTED** [review]
Thanks for adding this!

File: openverse_catalog/dags/common/extensions.py (hunk 1 above)
Comment (stacimc) [reply openverse_catalog/dags/common/extensions.py:10]: FWIW I think I'm convinced 😄 At any rate we can always iterate on that if we come up with a cleaner way to define the type. Better repeating ourselves once than in many places.

Comment (obulat) [reply openverse_catalog/dags/common/extensions.py:10]: I remember seeing the start of that discussion and have spent some time trying to find it in the code, so thank you for the link!
I'll just remove the type hints from these PRs, and we can implement it in other PRs :)

File: tests/dags/providers/provider_api_scripts/test_smk.py [hunk 2]
```diff
@@ -106,6 +114,33 @@ def test_handle_items_data_success():
     assert actual_image_count == 1
 
 
+def test_handle_items_data_success_data():
```
Comment (AetherUnbound) [inline tests/dags/providers/provider_api_scripts/test_smk.py:117]: Do we need to add the expected filesizes you added to the JSON files to other tests to assert they're being read in correctly? And do you mind adding a test case where filesize is missing to check that it gets set to `None` appropriately?

File: openverse_catalog/dags/common/extensions.py (hunk 1 above)
Comment (AetherUnbound) [reply openverse_catalog/dags/common/extensions.py]: I made an issue for it! https://github.com/WordPress/openverse-catalog/issues/563
